client/state/partner-portal/payment-methods/actions.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { AnyAction } from 'redux';
```

Comment:
What are the reasonsbehind defining the actions here compared to getting them from `createCreditCardPaymentMethodStore` (`client/my-sites/checkout/composite-checkout/payment-methods/credit-card/index.js`) ?

I'm asking just as much our of curiosity to get more familiar with React, stores and redux 😄 
Is it to avoid loading the entire store? And if so, could we abstract them out and make them importable separately so we don't have to maintain these 1:1 to extend the Blue checkout logic? Or would it just make more sense for us to keep them as is? 🧠 
If it's strictly to make them more easily testable, would it make sense to add the tests to the origin instead?

Reply:
The main reason for having a port instead of improving the existing one is that we would have to increase the complexity of the current composite-checkout logic to address both scenarios and them both are quite different. So instead of creating a huge one that is going to be loaded when not necessary to cover 2 not-so-equal scenarios, we decided to have separate ones that could be extended on their own if needed.